### DISCUSSION TITLE
Mute homebase as doorbell receiver

### DISCUSSION THREAD

**Discussion by Peertje87** [fuatakgun/eufy_security#505] [Ideas]
In the eufy app there is a possibility to set the homebase as doorbell receiver. When someone pushes the doorbell, the homebase will play a sound to notify. Would be great if we can automate this feature somehow. 

For example: 

- This doorbell can be a bit loud when my children are sleeping. Related to that it would be great to turn this feature off based on a time trigger.

**Comment by Brouwertje** [comment]
You can use volume. Mine goes to  1 when the shutter of kid is closed. 

![image](https://user-images.githubusercontent.com/89314233/201545564-51b86ee4-d2ee-4cce-8e93-f37f5e58e297.png)
